requirements.txt

```
@@ -1,8 +1,7 @@
 # pip3 install -U -r requirements.txt
 numpy
 scipy
-torch
-opencv-python
+cv2
```

❗ **CRITICAL**: `cv2` is not a valid PyPI package. Pip install from this requirements file will fail. Use `opencv-python` (or `opencv-python-headless` if GUI components are not needed) to provide the `cv2` module.

**Suggested change:**
```suggestion
opencv-python
```